molfeat/calc/fingerprints.py

```diff
@@ -139,58 +147,53 @@
     # COUNTING FP
     "ecfp-count": {
         "radius": 2,  # ECFP4
-        "nBits": 2048,
-        "invariants": [],
-        "fromAtoms": [],
-        "useChirality": False,
+        "fpSize": 2048,
+        "includeChirality": False,
         "useBondTypes": True,
-        "useFeatures": False,
-        "includeRedundantEnvironments": False,
+        "countSimulation": False,
+        "countBounds": None,
+        "atomInvariantsGenerator": None,
+        "bondInvariantsGenerator": None,
     },
     "fcfp-count": {
-        "radius": 2,  # FCFP4
-        "nBits": 2048,
-        "invariants": [],  # you may want to provide features invariance
-        "fromAtoms": [],
-        "useChirality": False,
+        "radius": 2,
+        "fpSize": 2048,
+        "includeChirality": False,
         "useBondTypes": True,
-        "useFeatures": True,
-        "includeRedundantEnvironments": False,
+        "countSimulation": False,
+        "countBounds": None,
+        "atomInvariantsGenerator": SerializableMorganFeatureAtomInvGen(),
```

Comment:
Can you confirm that the custom callable works ? Should cover that with a test if not done.

Reply:
Added a test case in `tests/test_fp.py`.